The drawer menu will filter between books and clothing when each is p…

jonathanopio opened the pull request:
…ressed, and all items shows all. Will work on the background color for the book and clothing pages..

jonathanopio committed:
The drawer menu will filter between books and clothing when each is pressed, and all items shows all. Will work on the background color for the book and clothing pages..

SaHendriksen commented:
There appear to be a fair number of issues with overflow and stuff being caused by this pull request. Maybe we can take a look at it in lab?

jonathanopio committed:
search delegate custom class

jonathanopio committed:
Add files via upload

SaHendriksen commented:
I think because this isn't the home screen, let's get rid of the collapsible header and do them just as a title on the app bar (see screens like Add Item). So just make it something meaningful like "All Items", "Clothing Items", etc. I think this will make it more clear to the user they are seeing a new screen.

SaHendriksen commented:
By getting rid of the collapsible header, you can implement something like we have on the Add Item screen which doesn't have a set height... I think this should get rid of the overflows you are getting.

jonathanopio committed:
Add files via upload

jonathanopio committed:
Add files via upload

"Fixed both filter pages to create an AppBar instead of a SliverAppBar, did the trick!"

SaHendriksen committed:
Merge branch 'master' into filter_item
